### PR TITLE
Resolves #VERAND-61 - Updated Android docs for SDK v.2.0.0

### PR DESCRIPTION
**Pull request by olllo123**

### CODE REVIEW

File: docs/verification/verification-for-android/verification-android-first-time-setup.md [hunk 1]
```diff
@@ -15,26 +15,52 @@ This is a step-by-step guide about setting up the Sinch Verification SDK for the
...
+
+Your application relies only on sms verification and doesn't use any other verificaiton methods. Then you should simply add:
+
+```text
+dependencies {
+    implementation 'com.sinch.android.sdk.verification:verification-sms:*.*.*'
+  }
+```
+
+The latest version of the SDK can be checked [here](https://bintray.com/sinch/com.sinch.android.sdk.verification).
```
Comment (Crozin) [inline docs/verification/verification-for-android/verification-android-first-time-setup.md:62]: Shouldn't we link to jcetner here? Also "...can be checked on jcenter.com" looks better than "...can be checked here".

File: docs/verification/verification-for-android/verification-android-first-time-setup.md [hunk 2]
```diff
@@ -15,26 +15,52 @@ This is a step-by-step guide about setting up the Sinch Verification SDK for the
...
+
+## Samples
 
-  - Or using Android Studio choose `File -> New -> New Module -> Import .JAR/.AAR Package` option
+A repository with fully functional samples is available on [GitHub](https://github.com/sinch/verification-samples) in _Android-Verification-SDK_ directory.
```
Comment (Crozin) [inline docs/verification/verification-for-android/verification-android-first-time-setup.md:66]: Just link to a concrete destination?

File: docs/verification/verification-for-android/verification-android-migration.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+title: The migration process
+excerpt: >-
+  The migration process
+---
+
+In order to migrate from version 1 to 2.\*.\* of the SDK one needs to change how Verification objects are created:
+
+```java
+val verification =
+  SinchVerification.create[Sms/Flashcall]Verification(config, phoneNumberInE164, listener)
```
Comment (Crozin) [inline docs/verification/verification-for-android/verification-android-migration.md:11]: Avoid "optional" code. Just provide a sample:

`val ver = SinchgVerification.createSmsVerification(...); // use .createFlashcall for flashcall`

**Review by Crozin — CHANGES_REQUESTED** [review]

File: docs/verification/verification-for-android/verification-android-first-time-setup.md (hunk 1 above)
Comment (olllo123) [reply docs/verification/verification-for-android/verification-android-first-time-setup.md:62]: jCenter is hosted by bintray so it jcenter "homepage" looks like this:
https://bintray.com/bintray/jcenter

http://jcenter.com/ has nothing to do with gradle, Android etc. I think a link to our bintray repository is fine as every public package on bintray will be linked to jCenter as well.

Comment (Crozin) [reply docs/verification/verification-for-android/verification-android-first-time-setup.md]: OK, I'm fine with that!

**Pull request merged**